planning/autoware_static_centerline_generator/launch/static_centerline_generator.launch.xml

```diff
@@ -55,7 +55,7 @@
   </node>
 
   <!-- optimize path -->
-  <node pkg="static_centerline_generator" exec="main" name="static_centerline_generator">
+  <node pkg="autoware_static_centerline_generator" exec="main" name="autoware_static_centerline_generator">
```

Comment:
```suggestion
  <node pkg="autoware_static_centerline_generator" exec="main" name="static_centerline_generator">
```
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/class-design/#gnss_poser_nodecpp

The default node name shouldn't have `autoware_` prefix.